src/engine/world/room.h

```diff
@@ -103,7 +105,10 @@ struct Room
 
   std::shared_ptr<render::scene::Node> node = nullptr;
   std::vector<gslu::nn_shared<render::scene::Node>> sceneryNodes{};
-
+  
+  std::map<uint8_t,std::shared_ptr<render::scene::Node>> dustCache{};
+  std::vector<loader::file::RoomVertex> dustVerticies{};
```

Comment:
`dustVerticies` is unnecessary - what's only required from this is a bounding box calculation.

What's only required is something like this:
```suggestion
    glm::vec3 verticesBBoxMin{std::numeric_limits<float>::max()};
    glm::vec3 verticesBBoxMax{std::numeric_limits<float>::lowest()};
```
These should be properly initialised in `createSceneNode`.